README.md

```diff
@@ -0,0 +1,37 @@
+Instance Identity module for Jenkins
+====================================
+
+This module maintains an RSA key pair that can serve
+as a foundation of authentication when communicating with Jenkins.
+
+## Description
+
+Each Jenkins instance maintains an RSA private/public key pair
+that can be used to uniquely identify Jenkins.
+This information is called "instance identity".
+
+From outside, the public key can be obtained by sending the GET request
+to the top page of Jenkins,
+and look for the _X-Instance-Identity_ header in the response.
+This header is always available, even if the response is 401 access denied
+(which can happen if Jenkins is protected via security.)
```

Comment:
The problem is the redirect to `/login` that doesn't have those headers (IIRC). Ask @rtyler some time how much he enjoyed figuring this out for the CLI docs: https://jenkins.io/doc/book/managing/cli/#ssh

Reply:
It was a copy-paste from Wiki. Pull requests are welcome

Reply:
Oops, missed that.